client/src/main/scala/com/dwolla/cloudflare/domain/model/accesscontrolrules/AccessControlRule.scala

```diff
@@ -0,0 +1,18 @@
+package com.dwolla.cloudflare.domain.model.accesscontrolrules
+
+case class Rule(
```

Comment:
Here's another spot where we could reformat—bring the `id: String` line up to the same line as `case class Rule(`, i.e.

```scala
case class Rule(id: String,
                notes: Option[String],
…
```